atmos_model.F90

```diff
@@ -2015,7 +2015,7 @@ subroutine assign_importdata(jdat, rc)
 
 ! get upward LW flux:  for sea ice covered area
 !----------------------------------------------
-          fldname = 'mean_up_lw_flx_ice'
+          fldname = 'lwup_flx_ice'
```

Comment:
I assume, the names without "mean" means inst fields?

Reply:
I could have picked a better alias, but yes. In general, I think the fields exchanged between components should simply being understood as the field value at the end of the model advance, the "instantaneous" value.